tidy3d/web/core/exceptions.py

```diff
@@ -10,3 +10,13 @@ def __init__(self, message: str = None):
         log = get_logger()
         super().__init__(message)
         log.error(message)
+
+
+class TaskRepeatedError(Exception):
+    """Repeated task in tidy3d"""
+
+    def __init__(self, message: str = None):
+        """Log just the error message and then raise the Exception."""
+        # log = get_logger()
+        super().__init__(message)
```

Comment:
I dont think this line of code does anything different from the original init. I think we can delete this whole method.